Security/Authorization/Voter/PageVoter.php

```diff
@@ -9,6 +9,11 @@
 class PageVoter implements VoterInterface
 {
     protected $container;
+    protected $onHold = false;
+
+    const PAGE_CLASS = 'Symbio\OrangeGate\PageBundle\Entity\Page';
+    const PAGE_ADMIN_CLASS = 'Symbio\OrangeGate\PageBundle\Admin\PageAdmin';
+    const PROXY_PAGE_CLASS = 'Proxies\__CG__\Symbio\OrangeGate\PageBundle\Entity\Page';
```

Comment:
Je tady vazne potreba pracovat s proxynou? Kde se tam vubec bere? Neni spatne to, ze se sem dostava proxy a ne skutecna page?